requirements.txt

```diff
@@ -1,7 +1,7 @@
-PyQt5>=5.15.9
+PyQt5==5.13.2
```

Comment:
hm, can we use different values for different linux builds?

Reply:
I believe we will have only one linux build and this version hopefully works for both linux and windows